packages/amplify-provider-awscloudformation/src/__tests__/utils/delete-from-parameter-store.test.ts

```diff
@@ -0,0 +1,35 @@
+import { $TSContext } from 'amplify-cli-core';
+import { getEnvParametersDeleteHandler } from '../../utils/delete-from-parameter-store';
+import { SSM } from '../../aws-utils/aws-ssm';
+
+jest.mock('../../aws-utils/aws-ssm');
+const mockSSM = SSM as jest.Mocked<typeof SSM>;
+const deleteParametersPromiseMock = jest.fn().mockImplementation(() => Promise.resolve());
+mockSSM.getInstance = jest.fn().mockResolvedValue({
+  client: {
+    deleteParameters: jest.fn().mockImplementation(() => ({
+      promise: deleteParametersPromiseMock,
+    })),
+  },
+});
+
+describe('parameters-delete-handler', () => {
+  it('check if returned function is defined', async () => {
```

Comment:
what happens if it's not defined?

Reply:
If it is not defined, then the parameters cannot be deleted from Parameter Store

Reply:
Sorry, poor wording. In this test, it is never possible for the returned function to be undefined. Once the test is reworked, the description of the test should match what is being asserted in the test.